Bearded.UI/Controls/implementations/ListControl.cs

```diff
@@ -95,11 +95,11 @@ public override void MouseScrolled(MouseScrollEventArgs eventArgs)
 
             if (offsetAfter > offsetBefore)
             {
-                onScrollDown();
+                onScrollDown(itemSource);
```

Comment:
Ok, I see what you mean now... there's not many checks, but the collection is passed around a whole lot indeed. I'm happy merging it like this, but perhaps for the future, I'd suggest we try refactoring all the private methods into a separate class wrapping the ItemSource internally, that has all the 'nullable behaviour' on it, so there's no passing and checks only at the entry points to 'private behaviour'. Though perhaps that'll end up with having to pull other fields into that class that we need in other places, and there's just no great solution here.